### DISCUSSION TITLE
We developed a huggingface_hub-like python library for civitai models' local management

### DISCUSSION THREAD

**Discussion by narugo1992** [civitai/civitai#634] [General]
As the title suggests, we have developed a library similar to huggingface_hub, which allows users to download and manage models from Civitai locally. You can find the library here: https://github.com/narugo1992/pycivitai .

This will be quite helpful for users with development or AI research needs, and it can be used in conjunction with various existing frameworks such as `diffusers`, `HCP-diffusion`, and more. 😄

**Comment by JustMaier** [comment]
Super cool! Looking forward to checking it out ❤️
